.github/actions/upgrade-indexer/upgrade-indexer.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Gets the absolute path of the script, used to load the common.sh file
+function get_absolute_path() {
+    ABSOLUTE_PATH="$( cd $(dirname ${0}) ; pwd -P )"
+}
+
+get_absolute_path
+. ${ABSOLUTE_PATH}/common.sh
+check_system
+check_version
+
+echo "Installing old version of Wazuh Indexer..."
+if [ ${sys_type} == "deb" ]; then
+    apt-get -y install wazuh-indexer
+elif [ ${sys_type} == "rpm" ]; then
+    preinstall_indexer_release
+    yum -y install wazuh-indexer
+else
+    echo "Error: No system detected"
+    exit 1
+fi
+
+read_files "${FILES_OLD}" "old"
+echo "Old files..."
+print_files "old"
+
+echo "Installing new version of Wazuh Indexer..."
+if [ ${sys_type} == "deb" ]; then
+    apt-get install $PACKAGE_NAME
+elif [ ${sys_type} == "rpm" ]; then
+    rpm -Uvh --nofiledigest $PACKAGE_NAME
```

Comment:
`yum install $PACKAGE_NAME` can't be used?

Reply:
It can be used, but adding the option `localinstall`. The complete command would be: `yum -y localinstall $PACKAGE_NAME`. Done.